charts/k8s-monitoring/ci/ci-2-values.yaml

```
@@ -76,8 +76,8 @@ test:
   - query: "count(kube_pod_info{cluster=\"ci-test-cluster-2\", color=\"blue\"})"
     type: promql
     expect:
-      operator: ">"
       value: 1
+      operator: "<="  # There's gotta be at least one pod
```

At least one, right?

```suggestion
      operator: ">="  # There's gotta be at least one pod
```

---

The check is `<expected> <operator> <actual>` so this will be computed as `1 <= (actual pod count)`